libraries/Zigbee/src/ZigbeeEP.cpp

```diff
@@ -33,16 +35,24 @@ void ZigbeeEP::setVersion(uint8_t version) {
 }
 
 bool ZigbeeEP::setManufacturerAndModel(const char *name, const char *model) {
+  // Allocate a new array of size length + 2 (1 for the length, 1 for null terminator)
+  // This memory space can't be deleted or overwritten, therefore using static
```

Comment:
@P-R-O-C-H-Y  - Could you please check if `esp_zb_basic_cluster_add_attr(...)` copies (duplicates) or just stores the pointer to both name strings?

Please let me know. Thanks!

Reply:
data is copied into a separated memory space.